lms/views/exceptions.py

```diff
@@ -13,83 +14,78 @@
 
 _ = i18n.TranslationStringFactory(__package__)
 
-DEFAULT_RENDERER = "lms:templates/error.html.jinja2"
 
+@view_defaults(path_info=not_("^/api/.*"), renderer="lms:templates/error.html.jinja2")
+class ExceptionViews:
```

Comment:
There's more work that can be done on these exception methods (for example a helper could reduce lines of code) but I've limited this PR to just moving them into a class

Reply:
Follow-ups: https://github.com/hypothesis/lms/pull/3162, https://github.com/hypothesis/lms/pull/3163, https://github.com/hypothesis/lms/pull/3164, https://github.com/hypothesis/lms/pull/3165